tools/building.py

```diff
@@ -38,11 +38,12 @@
 
 #  Building
 binaryen_checked = False
+EXPECTED_BINARYEN_VERSION = 109
 
-EXPECTED_BINARYEN_VERSION = 108
 # cache results of nm - it can be slow to run
 nm_cache = {}
 _is_ar_cache = {}
+
```

Comment:
Did you mean to add a newline here?

Reply:
Yes, I tried to group these in a slightly nicer way... if you disagree I can revert.